yearn/deploy/00_deploy_cvp_strategy.ts

```diff
@@ -0,0 +1,49 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, helpers } = hre
+  const { deploy, log } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const TBTCCurveVault = await deployments.getOrNull("TBTCCurveVault")
+  if (!TBTCCurveVault || !helpers.address.isValid(TBTCCurveVault.address)) {
+    throw new Error("Address of TBTCCurveVault must be set")
```

Comment:
Can we include `TBTCCurveVault.address` in the message?

Reply:
https://github.com/keep-network/tbtc-v2/pull/14/commits/fd4c1aed9986be5c8003bf17f3cd4200d8ac13d5